portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+/* IDs */
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#avatar {
+    width: 40%;
+}
+
+#title > * {
+  display: inline;
+}
+
+#title > a {
+  text-decoration: none;
+}
+
+#introduction{
+    background-color: #FAEBD7;
+}
+
+#education{
+    background-color: #F5DEB3;
+}
+
+#experience{
+    background-color: #CEB088;
+}
+
+#projects{
+    background-color: #967A76;
+}
+
+#fun-stuff{
+    background-color: #534C5A;
+    color: #534C5A;
+}
+
+#fun-stuff:hover{
+  color: white;
+}
+
+/* classes */
+.section {
+  display: inline-block;
+  padding: 1% 20% 1%;
```

Comment:
Optional improvement: 20% works for large desktop screens but not for small ones and probably not for mobile